src/js/DrawStyleController.js

```diff
@@ -54,131 +195,23 @@ goog.require('ga_styles_service');
-        {id: 'cafe'},
-        {id: 'cinema'},
-        {id: 'commercial'},
-        {id: 'clothing-store'},
-        {id: 'grocery'},
-        {id: 'fast-food'},
-        {id: 'hairdresser'},
-        {id: 'fuel'},
-        {id: 'laundry'},
-        {id: 'library'},
-        {id: 'lodging'},
-        {id: 'pharmacy'},
-        {id: 'restaurant'},
-        {id: 'shop'},
-
-        // Transport
-        {id: 'airport'},
-        {id: 'bicycle'},
-        {id: 'bus'},
-        {id: 'car'},
-        {id: 'ferry'},
-        {id: 'london-underground'},
-        {id: 'rail'},
-        {id: 'rail-above'},
-        {id: 'rail-light'},
-        {id: 'rail-metro'},
-        {id: 'rail-underground'},
-        {id: 'scooter'},
-
-        // Sport
-        {id: 'america-football'},
-        {id: 'baseball'},
-        {id: 'basketball'},
-        {id: 'cricket'},
-        {id: 'golf'},
-        {id: 'skiing'},
-        {id: 'soccer'},
-        {id: 'swimming'},
-        {id: 'tennis'},
-
-        // Places
-        {id: 'airfield'},
-        {id: 'building'},
-        {id: 'campsite'},
-        {id: 'cemetery'},
-        {id: 'city'},
-        {id: 'college'},
-        {id: 'dog-park'},
-        {id: 'embassy'},
-        {id: 'farm'},
-        {id: 'fire-station'},
-        {id: 'garden'},
-        {id: 'harbor'},
-        {id: 'heliport'},
-        {id: 'hospital'},
-        {id: 'industrial'},
-        {id: 'land-use'},
-        {id: 'lighthouse'},
-        {id: 'monument'},
-        {id: 'minefield'},
-        {id: 'museum'},
-        {id: 'oil-well'},
-        {id: 'park2'},
-        {id: 'park'},
-        {id: 'parking'},
-        {id: 'parking-garage'},
-        {id: 'pitch'},
-        {id: 'place-of-worship'},
-        {id: 'playground'},
-        {id: 'police'},
-        {id: 'polling-place'},
-        {id: 'post'},
-        {id: 'religious-christian'},
-        {id: 'religious-jewish'},
-        {id: 'religious-muslim'},
-        {id: 'prison'},
-        {id: 'school'},
-        {id: 'slaughterhouse'},
-        {id: 'theatre'},
-        {id: 'toilets'},
-        {id: 'town'},
-        {id: 'town-hall'},
-        {id: 'village'},
-        {id: 'warehouse'},
-        {id: 'wetland'},
-        {id: 'zoo'},
-
-        {id: 'camera'},
-        {id: 'chemist'},
-        {id: 'dam'},
-        {id: 'emergency-telephone'},
-        {id: 'entrance'},
-        {id: 'heart'},
-        {id: 'logging'},
-        {id: 'mobilephone'},
-        {id: 'music'},
-        {id: 'roadblock'},
-        {id: 'rocket'},
-        {id: 'suitcase'},
-        {id: 'telephone'},
-        {id: 'waste-basket'},
-        {id: 'water'}
+      // type: Type of icon: if stored as css style -> "css", if image ->"img"
+      // regex: pattern for icon category. Mandatory for category type "css"
+      iconCategories: [
+        {
+          id: 'standard',
+          label: 'standard',
+          useColorOption: true,
+          type: 'css',
+          regex: new RegExp('^(.*?)-24'),
```

Comment:
To simplify you can write directly `/^(.*)-24/` instead of `new Regexp(...) `   .
I also think th `? ` is not necesssary.